packages/camera/camera/pubspec.yaml

```diff
@@ -4,7 +4,7 @@ description: A Flutter plugin for controlling the camera. Supports previewing
   Dart.
 repository: https://github.com/flutter/packages/tree/main/packages/camera/camera
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+camera%22
-version: 0.10.5+2
+version: 0.10.5+3
```

Comment:
This is a new feature for a pre-1.0 plugin, so this should update the patch version. e.g. `0.10.6`.

Reply:
Changed to 0.10.6